src/styles/index.scss

```diff
@@ -1,3 +1,109 @@
-body {
+$main-color: #060b35;
+$secondary-color: #616070;
+
+* {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 0;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    padding: 0 3px 40px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-top: 4px;
+    margin-bottom: 16px;
+    color: $secondary-color;
+  }
+
+  &__review {
+    font-size: 10px;
+    line-height: 14px;
+    display: flex;
+    justify-content: space-between;
+    color: $main-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    line-height: 18px;
+    margin-top: 24px;
+
+    &_tag {
+      font-size: 12px;
+      font-weight: 400;
+      color: $secondary-color;
+    }
+
+    &_val {
+      font-size: 16px;
+      font-weight: 700;
+      color: $main-color;
+    }
+  }
+
+  &__buy-btn {
+    background-color: #00acdc;
+    height: 40px;
+    color: white;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 40px;
+    text-transform: uppercase;
+    text-decoration: none;
+    border-radius: 5px;
+    text-align: center;
+    margin: 16px 0;
+
+    &:hover {
+      box-sizing: border-box;
+      border: 1px solid #00acdc;
+      background-color: white;
+      color: #00acdc;
+    }
+  }
+}
+
+.stars {
+  box-sizing: border-box;
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+}
+
+@for $index from 0 through 5 {
+  .stars--#{$index} {
+    .stars__star:nth-child(-n + #{$index}) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
 }
```

Comment:
according to [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) there should be 1 file for each BEM block, please create stars file and move it here